airflow/providers/apache/cassandra/sensors/record.py

```diff
@@ -58,17 +63,19 @@ class CassandraRecordSensor(BaseSensorOperator):
     def __init__(
         self,
         *,
-        table: str,
         keys: Dict[str, str],
+        table: str,
         cassandra_conn_id: str = CassandraHook.default_conn_name,
         **kwargs: Any,
     ) -> None:
         super().__init__(**kwargs)
         self.cassandra_conn_id = cassandra_conn_id
+        if table is None:
+            raise RuntimeError("Please make sure that you set `table` parameter in the sensor")
```

Comment:
If the stub files work, this can be removed from the sensors. Fingers crossed.

Reply:
Ah yeah !

Reply:
It works nicely with the stubs! Great idea!